src/host/getset.cpp

```diff
@@ -1753,6 +1760,33 @@ void DoSrvSetCursorColor(SCREEN_INFORMATION& screenInfo,
     return Status;
 }
 
+// Routine Description:
+// - A private API call to get only the default text attributes of the screen buffer.

```

Comment:
(this may be resolved by comporting these with James' implementations)

the comment here doesn't seem totally accurate -- it's not acting on the _default_ attributes, is it?